### PR TITLE
Ensure correct target is used for ncc'ing web-vitals

### LINKED ISSUE #18910: Fix IE11 support with new ncc changes
IE11 fails with the latest ncc changes so we need to ensure the updates are compatible still!

### PR DESCRIPTION
**Pull request by ijjk**
This makes sure we don't use es6 syntax when compiling the `web-vitals` package with `ncc` since that breaks IE11 compatibility 

x-ref: https://github.com/vercel/ncc/pull/614
Closes: https://github.com/vercel/next.js/issues/18910

### CODE REVIEW

**Comment by ijjk** [comment]
## Failing test suites 
Commit: aca96c07daf168fc9852ff853e732a45b55c8324

**test/integration/api-catch-all/test/index.test.js** 
- API routes > Serverless support > should return data when catch-all
- API routes > Serverless support > should return redirect when catch-all with index and trailing slash
- API routes > Serverless support > should return data when catch-all with index and trailing slash
- API routes > Serverless support > should return data when catch-all with index and no trailing slash

<details>
<summary>Expand output</summary>

  ● API routes › Serverless support › should return data when catch-all

    command failed with code 1

      138 |         code !== 0
      139 |       ) {
    > 140 |         return reject(new Error(`command failed with code ${code}`))
          |                       ^
      141 |       }
      142 | 
      143 |       resolve({

      at ChildProcess.<anonymous> (lib/next-test-utils.js:140:23)

  ● API routes › Serverless support › should return redirect when catch-all with index and trailing slash

    command failed with code 1

      138 |         code !== 0
      139 |       ) {
    > 140 |         return reject(new Error(`command failed with code ${code}`))
          |                       ^
      141 |       }
      142 | 
      143 |       resolve({

      at ChildProcess.<anonymous> (lib/next-test-utils.js:140:23)

  ● API routes › Serverless support › should return data when catch-all with index and trailing slash

    command failed with code 1

      138 |         code !== 0
      139 |       ) {
    > 140 |         return reject(new Error(`command failed with code ${code}`))
          |                       ^
      141 |       }
      142 | 
      143 |       resolve({

      at ChildProcess.<anonymous> (lib/next-test-utils.js:140:23)

  ● API routes › Serverless support › should return data when catch-all with index and no trailing slash

    command failed with code 1

      138 |         code !== 0
      139 |       ) {
    > 140 |         return reject(new Error(`command failed with code ${code}`))
          |                       ^
      141 |       }
      142 | 
      143 |       resolve({

      at ChildProcess.<anonymous> (lib/next-test-utils.js:140:23)

</details>

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| buildDuration | 14.5s | 12.9s | -1.6s |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +4 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.177 | 2.148 | -0.03 |
| / avg req/sec | 1148.25 | 1163.69 | +15.44 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.225 | 1.216 | -0.01 |
| /error-in-render avg req/sec | 2041.42 | 2056.12 | +14.7 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..7d3b.js gzip | 11.3 kB | 11.3 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-82890a1..6613.js gzip | 7.4 kB | 7.4 kB | -4 B |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.4 kB | 58.4 kB | -4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 7.04 kB | 7.04 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-66aded3..dule.js gzip | 6.39 kB | 6.39 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.1 kB | 53.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-03d24c6..d1c7.js gzip | 1.35 kB | 1.35 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.75 kB | 7.75 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-dfbbfbd..dule.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.41 kB | 5.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 652 B | 652 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| index.html gzip | 1 kB | 1 kB | ✓ |
| link.html gzip | 1.01 kB | 1.01 kB | ✓ |
| withRouter.html gzip | 996 B | 996 B | ✓ |
| Overall change | 3.01 kB | 3.01 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>main-458f1bc..c8.module.js</strong></summary>

```diff
@@ -972,7 +972,7 @@
 
     /***/ p0hA: /***/ function(module, exports, __webpack_require__) {
       /* WEBPACK VAR INJECTION */ (function(__dirname) {
-        module.exports = (() => {
+        module.exports = (function() {
           var t = {
             692: function(t, e) {
               !(function(t, n) {
```
</details>
<details>
<summary>Diff for <strong>main-ab0ca6b..b4ce6a17d.js</strong></summary>

```diff
@@ -1314,7 +1314,7 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
 
     /***/ p0hA: /***/ function(module, exports, __webpack_require__) {
       /* WEBPACK VAR INJECTION */ (function(__dirname) {
-        module.exports = (() => {
+        module.exports = (function() {
           var t = {
             692: function(t, e) {
               !(function(t, n) {
```
</details>
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      href="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-ab0ca6b24c8b4ce6a17d.js"
+      src="/_next/static/chunks/main-dfdd84ebefd2e30bb51f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      src="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      href="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -90,13 +90,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-ab0ca6b24c8b4ce6a17d.js"
+      src="/_next/static/chunks/main-dfdd84ebefd2e30bb51f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      src="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      href="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-ab0ca6b24c8b4ce6a17d.js"
+      src="/_next/static/chunks/main-dfdd84ebefd2e30bb51f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      src="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| buildDuration | 13.3s | 13.1s | -164ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..7d3b.js gzip | 11.3 kB | 11.3 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-82890a1..6613.js gzip | 7.4 kB | N/A | N/A |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| main-a07ea88..1bad.js gzip | N/A | 7.4 kB | N/A |
| Overall change | 58.4 kB | 58.4 kB | -4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 7.04 kB | 7.04 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-66aded3..dule.js gzip | 6.39 kB | N/A | N/A |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| main-4cea1fc..dule.js gzip | N/A | 6.39 kB | N/A |
| Overall change | 53.1 kB | 53.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-03d24c6..d1c7.js gzip | 1.35 kB | 1.35 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.75 kB | 7.75 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-dfbbfbd..dule.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.41 kB | 5.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 652 B | 652 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _error.js | 914 kB | 914 kB | ✓ |
| 404.html | 4.73 kB | 4.73 kB | ✓ |
| hooks.html | 3.92 kB | 3.92 kB | ✓ |
| index.js | 914 kB | 914 kB | ✓ |
| link.js | 962 kB | 962 kB | ✓ |
| routerDirect.js | 956 kB | 956 kB | ✓ |
| withRouter.js | 956 kB | 956 kB | ✓ |
| Overall change | 4.71 MB | 4.71 MB | ✓ |

</details>

</details>
Commit: aca96c07daf168fc9852ff853e732a45b55c8324

**Comment by guybedford** [comment]
The failures here are the same failures I was getting for libraries that I needed to "un-ncc", so this does point to the root cause of https://github.com/vercel/ncc/issues/612 which I still haven't traced.

Is the option perhaps breaking the ncc build of web-vitals in some way? Have you tested the new build of it directly with the option?

File: yarn.lock [hunk 1]
```diff
@@ -3187,10 +3187,6 @@
   version "4.2.2"
   resolved "https://registry.yarnpkg.com/@xtuc/long/-/long-4.2.2.tgz#d291c6a4e97989b5c61d9acf396ae4fe133a718d"
 
-"@zeit/ncc@^0.20.4":
-  version "0.20.5"
-  resolved "https://registry.yarnpkg.com/@zeit/ncc/-/ncc-0.20.5.tgz#a41af6e6bcab4a58f4612bae6137f70bce0192e3"
```
Comment (guybedford) [inline yarn.lock:3192]: This might be interfereing with a stray `bin` reference somewhere. It could be worth trying this same PR with leaving this in.

**Comment by ijjk** [comment]
# Stats from current PR

<details>
<summary><strong>Default Server Mode</strong> (Increase detected ⚠️)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| buildDuration | 15.7s | 14.9s | -822ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +4 B |

</details>

<details>
<summary><strong>Page Load Tests</strong> Overall increase ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| / failed reqs | 0 | 0 | ✓ |
| / total time (seconds) | 2.872 | 2.846 | -0.03 |
| / avg req/sec | 870.62 | 878.51 | +7.89 |
| /error-in-render failed reqs | 0 | 0 | ✓ |
| /error-in-render total time (seconds) | 1.785 | 1.747 | -0.04 |
| /error-in-render avg req/sec | 1400.42 | 1430.66 | +30.24 |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..7d3b.js gzip | 11.3 kB | 11.3 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-82890a1..6613.js gzip | 7.4 kB | 7.4 kB | -4 B |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| Overall change | 58.4 kB | 58.4 kB | -4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 7.04 kB | 7.04 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-66aded3..dule.js gzip | 6.39 kB | 6.39 kB | ✓ |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| Overall change | 53.1 kB | 53.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-03d24c6..d1c7.js gzip | 1.35 kB | 1.35 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.75 kB | 7.75 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-dfbbfbd..dule.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.41 kB | 5.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 652 B | 652 B | ✓ |

</details>

<details>
<summary><strong>Rendered Page Sizes</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| index.html gzip | 1 kB | 1 kB | ✓ |
| link.html gzip | 1.01 kB | 1.01 kB | ✓ |
| withRouter.html gzip | 996 B | 996 B | ✓ |
| Overall change | 3.01 kB | 3.01 kB | ✓ |

</details>

#### Diffs
<details>
<summary>Diff for <strong>main-458f1bc..c8.module.js</strong></summary>

```diff
@@ -972,7 +972,7 @@
 
     /***/ p0hA: /***/ function(module, exports, __webpack_require__) {
       /* WEBPACK VAR INJECTION */ (function(__dirname) {
-        module.exports = (() => {
+        module.exports = (function() {
           var t = {
             692: function(t, e) {
               !(function(t, n) {
```
</details>
<details>
<summary>Diff for <strong>main-ab0ca6b..b4ce6a17d.js</strong></summary>

```diff
@@ -1314,7 +1314,7 @@ _N_E = (window["webpackJsonp_N_E"] = window["webpackJsonp_N_E"] || []).push([
 
     /***/ p0hA: /***/ function(module, exports, __webpack_require__) {
       /* WEBPACK VAR INJECTION */ (function(__dirname) {
-        module.exports = (() => {
+        module.exports = (function() {
           var t = {
             692: function(t, e) {
               !(function(t, n) {
```
</details>
<details>
<summary>Diff for <strong>index.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      href="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-ab0ca6b24c8b4ce6a17d.js"
+      src="/_next/static/chunks/main-dfdd84ebefd2e30bb51f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      src="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>link.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      href="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -90,13 +90,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-ab0ca6b24c8b4ce6a17d.js"
+      src="/_next/static/chunks/main-dfdd84ebefd2e30bb51f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      src="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
<details>
<summary>Diff for <strong>withRouter.html</strong></summary>

```diff
@@ -6,7 +6,7 @@
     <noscript data-n-css="true"></noscript>
     <link
       rel="preload"
-      href="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      href="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       as="script"
       crossorigin="anonymous"
     />
@@ -85,13 +85,13 @@
       src="/_next/static/chunks/polyfills-f73ba3fc145972ef83e9.js"
     ></script>
     <script
-      src="/_next/static/chunks/main-ab0ca6b24c8b4ce6a17d.js"
+      src="/_next/static/chunks/main-dfdd84ebefd2e30bb51f.js"
       async=""
       crossorigin="anonymous"
       nomodule=""
     ></script>
     <script
-      src="/_next/static/chunks/main-458f1bc1fe3898b5f5c8.module.js"
+      src="/_next/static/chunks/main-775e7e964eaf6ea70dd0.module.js"
       async=""
       crossorigin="anonymous"
       type="module"
```
</details>
</details>
<hr/>
<details>
<summary><strong>Serverless Mode</strong> (Decrease detected ✓)</summary>

<br/>

<details>
<summary><strong>General</strong> Overall increase ⚠️</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| buildDuration | 17.6s | 17.2s | -320ms |
| nodeModulesSize | 84.9 MB | 84.9 MB | ⚠️ +4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons)</strong> Overall decrease ✓</summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..7d3b.js gzip | 11.3 kB | 11.3 kB | ✓ |
| framework.HASH.js gzip | 39 kB | 39 kB | ✓ |
| main-82890a1..6613.js gzip | 7.4 kB | N/A | N/A |
| webpack-e067..f178.js gzip | 751 B | 751 B | ✓ |
| main-a07ea88..1bad.js gzip | N/A | 7.4 kB | N/A |
| Overall change | 58.4 kB | 58.4 kB | -4 B |

</details>

<details>
<summary><strong>Client Bundles (main, webpack, commons) Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| 677f882d2ed8..dule.js gzip | 7.04 kB | 7.04 kB | ✓ |
| framework.HA..dule.js gzip | 39 kB | 39 kB | ✓ |
| main-66aded3..dule.js gzip | 6.39 kB | N/A | N/A |
| webpack-07c5..dule.js gzip | 751 B | 751 B | ✓ |
| main-4cea1fc..dule.js gzip | N/A | 6.39 kB | N/A |
| Overall change | 53.1 kB | 53.1 kB | ✓ |

</details>

<details>
<summary><strong>Legacy Client Bundles (polyfills)</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| polyfills-4b..e242.js gzip | 31 kB | 31 kB | ✓ |
| Overall change | 31 kB | 31 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-7231d4b..5856.js gzip | 1.28 kB | 1.28 kB | ✓ |
| _error-fca3d..2eb1.js gzip | 3.44 kB | 3.44 kB | ✓ |
| hooks-d4591d..e7c2.js gzip | 887 B | 887 B | ✓ |
| index-17468f..5d83.js gzip | 227 B | 227 B | ✓ |
| link-03d24c6..d1c7.js gzip | 1.35 kB | 1.35 kB | ✓ |
| routerDirect..924c.js gzip | 284 B | 284 B | ✓ |
| withRouter-7..c13d.js gzip | 284 B | 284 B | ✓ |
| Overall change | 7.75 kB | 7.75 kB | ✓ |

</details>

<details>
<summary><strong>Client Pages Modern</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _app-75d3a82..dule.js gzip | 625 B | 625 B | ✓ |
| _error-4469a..dule.js gzip | 2.29 kB | 2.29 kB | ✓ |
| hooks-cbf13f..dule.js gzip | 387 B | 387 B | ✓ |
| index-b9a643..dule.js gzip | 226 B | 226 B | ✓ |
| link-dfbbfbd..dule.js gzip | 1.32 kB | 1.32 kB | ✓ |
| routerDirect..dule.js gzip | 284 B | 284 B | ✓ |
| withRouter-f..dule.js gzip | 282 B | 282 B | ✓ |
| Overall change | 5.41 kB | 5.41 kB | ✓ |

</details>

<details>
<summary><strong>Client Build Manifests</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _buildManifest.js gzip | 322 B | 322 B | ✓ |
| _buildManife..dule.js gzip | 330 B | 330 B | ✓ |
| Overall change | 652 B | 652 B | ✓ |

</details>

<details>
<summary><strong>Serverless bundles</strong></summary>

|  | vercel/next.js canary  | ijjk/next.js update/ncc-target | Change |
| - | - | - | - |
| _error.js | 914 kB | 914 kB | ✓ |
| 404.html | 4.73 kB | 4.73 kB | ✓ |
| hooks.html | 3.92 kB | 3.92 kB | ✓ |
| index.js | 914 kB | 914 kB | ✓ |
| link.js | 962 kB | 962 kB | ✓ |
| routerDirect.js | 956 kB | 956 kB | ✓ |
| withRouter.js | 956 kB | 956 kB | ✓ |
| Overall change | 4.71 MB | 4.71 MB | ✓ |

</details>

</details>
Commit: 61891ba81d3eb36b016843ac0fba0473a4f74b54

**Review by guybedford — APPROVED** [review]

**Review by Timer — APPROVED** [review]

**Pull request merged**